newdle/client/src/components/Title.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Container, Header} from 'semantic-ui-react';
+import styles from './Title.module.scss';
+export default function Title() {
```

Comment:
I think if we go for a component name like this we absolutely need to start moving stuff to a subdirectory, e.g. `components/answer/Title.js`.